firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -232,17 +242,32 @@ export class FirestoreBigQueryEventHistoryTracker
       const documentIdColExists = fields.find(
         (column) => column.name === "document_id"
       );
+      const paramsColExists = fields.find(
+        (column) => column.name === "path_params"
+      );
 
-      if (!documentIdColExists) {
+      if (!documentIdColExists && this.config.wildcardIds) {
```

Comment:
Is this check `&& this.config.wildcardIds` in the correct place.

Could it be expected to alongside `paramsColExists`?